src/gui/ProductDataHandler.hpp

```diff
@@ -2,14 +2,16 @@
 
 #include <QObject>
 
+#include <core/Core.hpp>
```

Comment:
dla plików projektowych używamy "", dla zewnętrznych <> 

https://stackoverflow.com/questions/21593/what-is-the-difference-between-include-filename-and-include-filename

Reply:
ok, poprawione
Takie samo nawiasowanie jest w src/main.cpp, więc też będzie do zmiany https://github.com/mnurzyns/BakcylProgramowania2021/issues/56